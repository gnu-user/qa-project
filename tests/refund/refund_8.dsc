TITLE
    Refund Amount Input (Invalid input, garbage)
DESCRIPTION
    Run the login transaction using an admin account. Initiate the refund 
    transaction. When prompted, enter a buyer which exists. When prompted, 
    enter a seller which exists. When prompted, enter a garbage non-numeric 
    value for the refund amount. Verify that the value is rejected and an error 
    occurs.
DEPENDENCIES
    refund_7
