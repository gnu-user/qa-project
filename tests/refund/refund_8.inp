login
administrator
refund
buyer
seller
invaliddollars