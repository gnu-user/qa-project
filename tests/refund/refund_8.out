Welcome.
> 
Enter username: 
Username accepted.
Available tickets file read successfully.
>
Enter the buyer's username: 
Enter the seller's username: 
Enter the amount of credit to transfer: 
Invalid dollars value, amount must be in dollars (e.g. 10.00).